Nsubjettiness/MeasureDefinition.cc

```diff
@@ -383,9 +383,9 @@ std::vector<LightLikeAxis> DefaultMeasure::UpdateAxesFast(const std::vector <Lig
                                                           ) const {
    assert(old_axes.size() == N);
    
-   // some storage, declared static to save allocation/re-allocation costs
-   static LightLikeAxis new_axes[N];
-   static fastjet::PseudoJet new_jets[N];
+   // some storage, declared on stack to save allocation/re-allocation costs
```

Comment:
I had interpreted the comment to be relative to new and delete. Looks to me like the following for loop could also be removed now.